homeassistant/components/deconz/fan.py

```diff
@@ -156,6 +168,7 @@ def percentage_to_speed(self, percentage: int) -> str:
     @property
     def supported_features(self) -> int:
         """Flag supported features."""
+        assert self._attr_supported_features
```

Comment:
Fans do not support shorthand property implementation:

<https://developers.home-assistant.io/docs/core/entity/fan>

It should not be asserted, it should be removed from the fan implementation instead.

Reply:
Reverted this change

Reply:
Meanwhile, I've just merged a contribution adding shorthand support for the Fan entity 🎉

Reply:
Added it back again :)